frontend/views/utils/stepAssistant.js

```diff
@@ -18,26 +18,35 @@
 export default {
   name: 'StepAssistant',
   mounted () {
-    if (this.currentStep === 0) {
-      this.$router.push({ name: this.config.steps[this.currentStep] })
-    }
+    this.redirect(this.config.steps[this.currentStep])
```

Comment:
My only concern with step assistant: We do the redirect to the correct step, but all the previous form data is lost... So, what's the point of doing the redirect if there's no data? Maybe, for now, we should disable that and only activate it when we "recover" the form data.

Reply:
I don't think so. If we want to keep the form data, that's a feature we should implement. Please don't block unnecessary the PR

Reply:
I agree with you, we need to implement it in another task, so, for now, the only thing I was requesting is to disable the redirect until we implement the form data recover.

Reply:
I disagree. This is removing a functionality which is useful for testing and for the end user sharing the screen. It doesn't block the user because the validation stop the user if data is missing and user can go back to fill the missing steps